Spanish (Spain) translation

treborin opened the pull request.

datamweb commented:
Hey @treborin ,
Thank you for contributing!
Please see https://github.com/codeigniter4/shield/pull/235#issuecomment-1153050805

treborin commented:
> Hey @treborin , Thank you for contributing! Please see [#235 (comment)](https://github.com/codeigniter4/shield/pull/235#issuecomment-1153050805)

@datamweb . Done!

datamweb commented on tests/Language/AbstractTranslationTestCase.php:
```diff
@@ -49,7 +49,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         //        CzechTranslationTest::class              => 'cs',
         GermanTranslationTest::class => 'de',
-        //        SpanishTranslationTest::class            => 'es',
+        SpanishTranslationTest::class                      => 'es',
```
Please fix the coding style. **just run :**
`composer cs-fix`

datamweb commented:
It looks like you are using GitHub UI to post this PR.
Please rename file `tests/Language/SpanishTranslationTest` to `tests/Language/SpanishTranslationTest.php`.
Then tell me if there is any problem in executing command `composer cs-fix`?

datamweb commented:
> ![image](https://user-images.githubusercontent.com/31905622/183878098-de0216bd-6fd7-40b0-ab61-c1d202e045d9.png)

@kenjis , Does executing command `composer update` solve this problem?

kenjis commented:
@treborin run `composer update` and `composer cs-fix` in the repository root directory, that is `shield`.

treborin commented:
> @treborin run `composer update` and `composer cs-fix` in the repository root directory, that is `shield`.

@kenjis , cmposer update, ok. But here is composer cs-fix:
![image](https://user-images.githubusercontent.com/31905622/183908489-47498edd-0e1b-4043-a9cd-c42d3c8155e8.png)

@datamweb, i fixed coding style by hand. I saw inside unsuccessfuls checks thay te problem was the file text format, that need to be ansii. And i fixed it by hand, line by line.

datamweb commented:
Yes, I got it.
try 
`mkdir build`
`composer update` 
`composer cs-fix`

treborin commented:
> Yes, I got it. try `mkdir build` `composer update` `composer cs-fix`

![image](https://user-images.githubusercontent.com/31905622/183914539-57d11fb7-e832-4891-8b30-774721b3f1a3.png)
![image](https://user-images.githubusercontent.com/31905622/183914862-116388da-7b0e-4215-957e-b24d0459b1d5.png)

MGatner commented:
The `cs-fix` command was just added this week (#340). If your fork was from earlier the command would be `composer style`. Either way that is simply an alias, you can run the command directly:
```shell
php vendor/bin/php-cs-fixer fix --ansi
```

MGatner commented:
I don't know what is happening with those action failures. The tests all pass but a PHP error occurs when it tries to gather coverage. I believe it is unrelated to this PR.

treborin commented:
> I don't know what is happening with those action failures. The tests all pass but a PHP error occurs when it tries to gather coverage. I believe it is unrelated to this PR.

I try with a new PR
